### PR TITLE
Update committee page for 23-24

### LINKED ISSUE #416: Update committee page with 23-24 committee

### PR DESCRIPTION
**Pull request by MattyTheHacker**
- [x] President
- [x] Treasurer
- [x] Secretary
- [x] Vice President
- [x] Publicity
- [x] International
- [x] Special Events
- [x] Social
- [x] Sports
- [x] EDI
- [x] Industrial
- [x] PGT
- [x] PGR
- [x] First Year

closes #416

### CODE REVIEW

File: _scripts/committee.sh
Comment (MattyTheHacker) [inline _scripts/committee.sh:17]: Not that big of a deal at all but just as a note for the future this change should probably be it's own separate PR.

**Review by codemicro — DISMISSED** [review]

**Review by CharlesESW — APPROVED** [review]
I like this, but don’t merge until the 1st, as that is official handover

**Review by codemicro — APPROVED** [review]
Once again, LGTM but don't merge until the 1st

**Review by Thatsmusic99 — APPROVED** [review]
Its mergin time

**Comment by codemicro** [comment]
it has happened

**Pull request merged**
